### DISCUSSION TITLE
Any one else having problems with date [Aug 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6174] [Discord Threads]
> 🙋‍♂️ **victorminemu**

**Comment by o1lab** [comment]
> 🙋‍♂️ **evalshrew**  
#### Like `where=(CreatedAt,eq,2023-07-04 00:38:03+00:00)`?   
####    
#### I've literally just come here with the same problem if so. I'm hitting a 500 when I include this where query  
#  
> 🙋‍♂️ **victorminemu**  
#### I've fixed it but is for the UI that only lets you select date and not the time <@235778360167497728>  
#  
> 🙋‍♂️ **victorminemu**  
#### If we want to have the possibility to search by date and time we need a condition for exact date  
#  
> 🙋‍♂️ **victorminemu**  
#### Both for GUI & API  
#  
> 🙋‍♂️ **victorminemu**  
#### <@235778360167497728> try this ver: https://github.com/nocodb/nocodb/pull/6017  
#  
> 🙋‍♂️ **evalshrew**  
#### Ah I've actually dug into this further and discovered that these both work   
####    
#### `(created_at,btw,2023-07-04 00:38:03,2023-07-04 23:38:03)`   
#### `(created_at,eq,2023-07-04 00:38:03)`   
####    
#### I think it's about the column name mapping, "CreatedAt" is used in the interface, but the column in the DB is `created_at`  
#  
> 🙋‍♂️ **evalshrew**  
#### Bug ticket for this https://github.com/nocodb/nocodb/issues/6023  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1126523502032343070)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
